express-notes-api/index.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const data = require('./data.json');
+var arr = [];
```

Comment:
This array should only be created inside your `/api/notes` only. That way each `GET /api/notes` generates a fresh array.